src/components/crossDevice/MobileComplete/style.css

```diff
@@ -0,0 +1,47 @@
+.title {
+  height: auto;
+  margin-bottom: 16px;
+}
+
+.submessage {
+  margin-bottom: 48px;
+}
+
+.icon {
+  height: 36px;
+  width: 36px;
+  float: left;
+  background-size: contain;
+  background-image: url('assets/complete-tick-small.svg');
+}
+
+.uploadList {
+  height: 112px;
+  display: inline-block;
+  margin: auto;
+}
+
+.uploadList li {
+  list-style: none;
+}
+
+.listItem {
+  height: 24px;
+  width: auto;
+  color: #585E5F;
+  float: left;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 36px;
+  margin-bottom: 40px;
+  margin-left: 14px;
+}
+
+.lastItem {
```

Comment:
Minor: repeated selector

Reply:
No, one is `lastItem` and the other one is `listItem`. Maybe I should name it differently 😄